csi/server/plugin/opensds/controller.go

```diff
@@ -125,6 +126,8 @@ func (p *Plugin) CreateVolume(
 	var enableReplication = false
 	for k, v := range req.GetParameters() {
 		switch strings.ToLower(k) {
+		case "fstype":
```

Comment:
Can you use the constant defined instead?

Reply:
Already modified, please review again.